web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -0,0 +1,82 @@
+from impact.v1.helpers.model_helper import ModelHelper
+
+VALID_GENDERS = ["f", "m", "o", "p"]
+
+GENDER_TRANSLATIONS = {
+    "female": "f",
+    "male": "m",
+    "other": "o",
+    "prefer not to state": "p",
+}
+
+INVALID_GENDER_ERROR = ("Invalid gender: '{}'. Valid values are "
+                        "'f' or 'female', 'm' or 'male', "
+                        "'o' or 'other', and 'p' or 'prefer not to state'")
+
+
+class ProfileHelper(ModelHelper):
+    REQUIRED_KEYS = [
+        "gender",
+        ]
+    OPTIONAL_STRING_KEYS = [
+        "bio",
+        "company",
+        "facebook_url",
+        "linked_in_url",
+        "office_hours_topics",
+        "personal_website_url",
+        "phone",
+        "referred_by",
+        "speaker_topics",
+        "title",
+        "twitter_handle",
+        ]
+    OPTIONAL_BOOLEAN_KEYS = [
+        "judge_interest",
+        "mentor_interest",
+        "office_hours_interest",
+        "speaker_interest",
+        ]
+    OPTIONAL_KEYS = OPTIONAL_BOOLEAN_KEYS + OPTIONAL_STRING_KEYS
+    INPUT_KEYS = REQUIRED_KEYS + OPTIONAL_KEYS
+
+    READ_ONLY_KEYS = [
+        "additional_industry_ids",
+        "expert_category",
+        "mentoring_specialties",
+        "primary_industry_id",
+        "updated_at",
+        ]
+    OUTPUT_KEYS = READ_ONLY_KEYS + INPUT_KEYS
+
+    @property
+    def additional_industry_ids(self):
+        return self.subject.additional_industry_expertise.values_list(
+            "id", flat=True)
+
+    @property
+    def primary_industry_id(self):
+        return self.subject.primary_industry_expertise_id
+
+    @property
+    def expert_category(self):
+        if hasattr(self.subject, "expert_category"):
+            category = self.subject.expert_category
+            if category:
+                return category.name
+
+    @property
+    def mentoring_specialties(self):
+        if hasattr(self.subject, "mentoring_specialties"):
+            specialties = self.subject.mentoring_specialties
+            if specialties:
+                return [specialty.name for specialty in specialties.all()]
+
+
+def find_gender(gender):
+    if not hasattr(gender, "lower"):
```

Comment:
shouldnt we be checking that its a string instead?
```if type(gender) == str```

Reply:
This seems like a bit of philosophical question.  My check is explicitly checking that the next line will succeed while accepting as rich a range of potential input as possible.  Your approach restricts the type of the argument to strings which currently are the only values we pass in, so practically there is no different. I would argue that my approach is more future proof.  In any case if we're going your way then really we should be doing isinstance(gender, str).  Obviously I have a slight preference for my approach, but I'm willing to make the change to isinstance if it's important to you.

Reply:
I'm going to be a stickler on the string check because I think the context for the next set of instructions is reliant on it being a string and not just having a ```lower``` method defined - Also curious as to why this was added; are there places we expect this to be called with non-string inputs?

Reply:
It's defensive coding habit because we have no control over what we might get from the user.

Reply:
Done.